codedigger/codeforces/tests/test_api.py

```diff
@@ -12,6 +12,12 @@ def test_user_info_api(self):
         self.assertEqual(len(response), 2)
         self.assertEqual(response[0]['handle'], 'shivamsinghal1012')
 
+    def test_user_rating_api(self):
+        handle = 'Fefer_Ivan'
+        response = user_rating(handle)
+        self.assertEqual(len(response), 85)
```

Comment:
You can't use len(response) while testing, as this is a variable and whenever user gave new contest, this value will change.

Reply:
Yeah, testing for exactly 85 elements was bad. 
I changed it to greater than 0.